moclojer.edn

```diff
@@ -0,0 +1,7 @@
+{:type :edn
+ :endpoints {:get-all-pets {:method :get
```

Comment:
`:endpoints`: in `yaml` we use the name in singular (not plural) and identify it as a list.
Can we do that in edn?

This way we keep the naming standard for all kinds of configuration files

```yaml
- endpoint:
    # Note: the method could be omitted because GET is the default
    method: GET
```

Reply:
I changed the edn file, I think makes senses follow the same pattern as yaml file